static/docs/commands-reference/status.md

```diff
@@ -56,7 +56,12 @@ differ are listed. For each item listed, either the file name or the checksum is
 shown, and additionally a status word is shown describing the change:
 
 - For the local workspace:
-  - _changed_ means the named file has changed
+  - _changed deps_ means the named file has changed dependencies
+  - _changed outs_ means the named file has changed outputs
+  - _changed checksum_ means actual check sum of data dosen't match specified in
+    dvc files
+  - _always checksum_ means special dvc file with no dependencies, considered
+    always changed
```

Comment:
@efiop could you take a look please? do we mention all possible variants now?

Reply:
We are missing statuses for the out/dep files themselves. E.g. 
```
not in cache
deleted
modified
new
```

Reply:
@efiop shall I add these statues too in the description?

Reply:
@sanidhyamangal Yes, please :slightly_smiling_face: